lib/ledger_sync/adaptors/ledger_serializer_attribute.rb

```diff
@@ -30,6 +44,10 @@ def block_value_for(resource:)
         block.call(resource)
       end
 
+      def deserialize?
```

Comment:
I thought you got these for free. Only in rails land?

Reply:
Yeah. :(